### PR TITLE
feat(authentication-service): adds fullName property in user model

### PR DESCRIPTION
**Pull request by vineet-suri**
SFO-165

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: services/authentication-service/package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
   ],
   "dependencies": {
     "@loopback/boot": "^2.3.1",
-    "@loopback/context": "^3.9.2",
+    "@loopback/context": "^3.13.0",
```
Comment (samarpan-b) [inline services/authentication-service/package.json:46]: this will cause issues. Please revert this change for now.

**Review by samarpan-b — CHANGES_REQUESTED** [review]

**Comment by vineet-suri** [comment]
re done in https://github.com/sourcefuse/loopback4-microservice-catalog/pull/98
